scipy/stats/stats.py

```diff
@@ -2858,13 +2893,111 @@ def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
         med = np.apply_over_axes(center, arr, axis)
         mad = np.median(np.abs(arr - med), axis=axis)
 
-    return scale * mad
+    # NOTE: _scale_conversions was for IQR so for MAD we need to divide by 2
+    if is_str and scale_key == 'normal':
+        scale /= 2
+
+    return mad / scale
+
 
+_median_absolute_deviation_deprec_msg = """\
+The function `median_absolute_deviation` is deprecated.
+To preserve the existing behavior of the program, use
+`scipy.stats.median_abs_deviation(..., scale=1.4826)`.
+The value 1.4826 is not numerically precise for scaling
+with a normal distribution. For a numerically precise value, use
+`scipy.stats.median_abs_deviation(..., scale='normal')`.
+."""
+
+
+@np.deprecate(old_name='median_absolute_deviation',
+              new_name='median_abs_deviation',
+              message=_median_absolute_deviation_deprec_msg)
+def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
+                              nan_policy='propagate'):
+    """
+    Compute the median absolute deviation of the data along the given axis.
+    The median absolute deviation (MAD, [1]_) computes the median over the
+    absolute deviations from the median. It is a measure of dispersion
+    similar to the standard deviation but more robust to outliers [2]_.
+    The MAD of an empty array is ``np.nan``.
+    .. versionadded:: 1.3.0
+    Parameters
+    ----------
+    x : array_like
+        Input array or object that can be converted to an array.
+    axis : int or None, optional
+        Axis along which the range is computed. Default is 0. If None, compute
+        the MAD over the entire array.
+    center : callable, optional
+        A function that will return the central value. The default is to use
+        np.median. Any user defined function used will need to have the function
+        signature ``func(arr, axis)``.
+    scale : int, optional
+        The scaling factor applied to the MAD. The default scale (1.4826)
+        ensures consistency with the standard deviation for normally distributed
+        data.
+    nan_policy : {'propagate', 'raise', 'omit'}, optional
+        Defines how to handle when input contains nan.
+        The following options are available (default is 'propagate'):
+          * 'propagate': returns nan
+          * 'raise': throws an error
+          * 'omit': performs the calculations ignoring nan values
```

Comment:
Needs newline, etc -- lots of newlines missing in this docstring

Reply:
yes, I see now, before `* 'propagate'` line